### PR TITLE
June25 ctl

### PR DESCRIPTION
**Pull request by chris-little**
English improvements, consistency of naming, consistent with OGC Naming policy (except resourceName plural)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ It will also contain the plan of work and links to other relevant documents such
...
+The EDR API can be considered a 'Sampling API'. EDR queries create discrete sampling geometries that can sample a relatively persistent data store resource. The query and its response are transient resources, which can be made persistent for re-use if required. EDR is agnostic to whether the data store is a digital data cube that could be sampled anywhere or pre-existing samples of, or a model of, real world phenomena. While the former is the emphasis, EDR APIs can provide a list of pre-defined or pre-existing monitoring/modeled "locations" which can be accessed by location identifier. There is an assumption that the data store is non-sparse, in that most queries are expected to return useful values rather than 'data not found'.
```
Comment (m-burgoyne) [inline README.md:24]: Is it worth changing: *There is an assumption that the data store is non-sparse* to something along the lines of  *There is an assumption that the data store is non-sparse or the service is capable of providing an interpolated result*

**Comment by chris-little** [comment]
I support your proposed wording re interpolation possibility

File: candidate-standard/EDR-candidate-specification.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-:Title: OGC API - Environmental Data Retrieval Standard - Part 1 - Core
+:Title: OGC API - Environmental Data Retrieval Standard
```
Comment (dblodgett-usgs) [inline candidate-standard/EDR-candidate-specification.adoc:1]: Need to finalize this in #86

File: candidate-standard/clause_5_conventions.adoc [hunk 3]
```diff
@@ -48,6 +48,8 @@ The following link-relations are in common use by OGC Web API Standards.
 
 Each resource representation includes an array of links. Implementations are free to add additional links for all resources provided by the API. For example, an *enclosure* link could reference a bulk download of a collection. Or a *related* link on a feature could reference a related feature.
 
+*Note*: The query patterns of the EDR API use the link relation *data*. It is envisaged that, in the future, this link relation may be replaced by *position*, *area*, and *trajectory* which would all be specializations of the currently used *data*.
+
```
Comment (dblodgett-usgs) [inline candidate-standard/clause_5_conventions.adoc:52]: #72

File: candidate-standard/clause_8_collections.adoc [hunk 4]
```diff
@@ -2,9 +2,9 @@
 === Query Resources
 :sectnums:
 
-Query Resources are spatial queries which support the operation of the API or the access and use of the Environmental Resources. They are described in the <<query-resources>> section.
+Query Resources are spatial queries which support the operation of the API or the access and use of the Environmental Data Resources. They are described in the <<query-resources>> section.
```
Comment (dblodgett-usgs) [inline candidate-standard/clause_8_collections.adoc:5]: Need to revisit "Environmental Data Resources" in the context of #86

**Pull request merged**